halibot/halobject.py

```diff
@@ -42,21 +42,33 @@ def init(self):
 	def shutdown(self):
 		pass
 
+	def raw_send(self, msg):
+		if not msg.target:
+			self.log.warning("Message passed to send without target")
```

Comment:
Should we raise an exception here? (and log)

Reply:
Hmmmmm... probably. We should probably be raising more exceptions in general than we are, rather than silently failing. Since `.raw_send()` should be only be sparingly directly called, I think it's even more important that we complain loudly about malformed `msg`s.